File: VB/Models/Document.vb
Imports System.Windows

Namespace DXRibbonNotepad.Model

    Public Class Document

        Public Property Text As String

        Public Property DocumentName As String

        Public Property IsBold As Boolean

        Public Property IsItalic As Boolean

        Public Property IsUnderline As Boolean

        Public Property TextAlignment As TextAlignment

        Public Property FontSize As Double

        Public Property FontFamily As String

        Public Sub New()
            Text = String.Empty
            DocumentName = "Untitled"
            FontSize = 12R
            FontFamily = "Consolas"
        End Sub
    End Class
End Namespace

File: CS/Models/Document.cs
using System.Windows;
namespace DXRibbonNotepad.Model {
    public class Document {
        public string Text { get; set; }
        public string DocumentName { get; set; }
        public bool IsBold { get; set; }
        public bool IsItalic { get; set; }
        public bool IsUnderline { get; set; }
        public TextAlignment TextAlignment { get; set; }
        public double FontSize { get; set; }
        public string FontFamily { get; set; }
        public Document() {
            Text = string.Empty;
            DocumentName = "Untitled";
            FontSize = 12d;
            FontFamily = "Consolas";
        }
    }
}